README.md

```diff
@@ -337,6 +335,35 @@ func main() {
 For more descriptive exception types, please see the [Twilio documentation](https://www.twilio.com/docs/libraries/go/usage-guide#exceptions).
 
 ## Advanced Usage
+### Using Standalone Products
+Don't want to import the top-level Twilio RestClient with access to the full suite of Twilio products? Use standalone product services instead:
+```go
```

Comment:
When I try to run this example, I'm getting:

```
./main.go:19:15: defaultClient.SetAccountSid undefined (type *client.Client has no field or method SetAccountSid)
./main.go:21:20: undefined: "github.com/twilio/twilio-go/rest/api/v2010".NewDefaultApiServiceWithClient
./main.go:22:26: undefined: "github.com/twilio/twilio-go/rest/serverless/v1".NewDefaultApiServiceWithClient
```

Reply:
Are you using the main branch or pulling a released version? These changes haven't been released yet.

Reply:
Ah, I think I'm pulling the released version v0.8.0. Will test against main. Thanks!